app.py

```diff
@@ -270,16 +275,25 @@ def _create_firefox():
     options.set_preference("general.useragent.override",
                            "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:96.0) "
                            "Gecko/20100101 Firefox/96.0")
-    if _HEADLESS:
+    if headless:
         options.headless = True
 
     cap = DesiredCapabilities.FIREFOX
     cap["marionette"] = False
 
-    driver = webdriver.Firefox(options=options, desired_capabilities=cap)
+    driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()), options=options,
+                               desired_capabilities=cap)
     return driver
 
 
+@app.route('/health_check', methods=['get'])
+def health_check():
+    return{
+        "DISPLAY": os.getenv('DISPLAY'),
```

Comment:
This may be a security risk, please remove it

Reply:
Do you want me to return a string instead the parameters like "ok", or remove the endpoint?